docs/tutorials/tokens/erc1155.md

```diff
@@ -183,7 +181,7 @@ Here is an example request that would let the signing account `0x634ee8c7d0894d0
   "approved": true,
   "operator": "0x634ee8c7d0894d086c7af1fc8514736aed251528",
   "config": {
-      "amount": "10000000000000000000"
+      "allowance": "10000000000000000000"
```

Comment:
No allowance supported by ERC-1155. Also looks like you've duplicated the response body twice.